DeepCat.js

```diff
@@ -166,14 +209,15 @@
 			newSearchTerms[userParameters['searchTermNum']] = newSearchTermString;
 		}
 
+		DeepCat.ResponseErrors.addError(errorMessages);
```

Comment:
Shouldn't this be errorMessages[0] ? You could probably refactor errorMessages here due to the fact, that in this place you only have one case were you create one error message. You could add a conditional clause as well if adding an error is even needed.